src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -212,10 +212,35 @@ protected ConstantValue MakeDefaultExpression(DiagnosticBag diagnostics, Binder
                 }
             }
 
+            if (parameterType.IsNullable == false &&
```

Comment:
I believe that when LangVersion = 7, `IsNullable` returns `null` (and that's how we avoid producing a warning). Is that right?

Reply:
That's correct.

---
In reply to: [186878903](https://github.com/dotnet/roslyn/pull/26726#discussion_r186878903) [](ancestors = 186878903)